pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -85,17 +153,31 @@ stdenv.mkDerivation rec {
   cmakeDir = "../cmake";
 
   cmakeFlags = [
-    "-Donnxruntime_PREFER_SYSTEM_LIB=ON"
+    "-DCMAKE_BUILD_TYPE=RELEASE"
+    "-DFETCHCONTENT_FULLY_DISCONNECTED=ON"
+    "-DFETCHCONTENT_QUIET=OFF"
+    "-DFETCHCONTENT_SOURCE_DIR_ABSEIL_CPP=${abseil-cpp_202206.src}"
+    "-DFETCHCONTENT_SOURCE_DIR_DATE=${howard-hinnant-date}"
+    "-DFETCHCONTENT_SOURCE_DIR_EIGEN=${eigen}"
```

Comment:
It also builds fine for me with the nixpkgs version with `eigen.src`.

Reply:
I have seen this error before. ONNX Runtime 1.15.0 will be released in next week. It will use eigen 3.4.0.  But I don't see this build error anymore. I am also curious what gets changed.

Reply:
It seems that that this change did not make it into 1.15.0 after all, did it, @snnn?

Reply:
You are right.

Reply:
Is there one or two patches we can fetch to fix this? We don't want to vendor.

Reply:
No, updating the dependencies is a major undertaking by the looks of it.